src/table/use-sticky-scrollbar.ts

```diff
@@ -90,7 +90,10 @@ export function useStickyScrollbar(
   // containing block, if present, is below the app layout and above the overflow
   // parent, which is a pretty safe assumption.
   const [hasContainingBlock, setHasContainingBlock] = useState(false);
-  const consideredFooterHeight = hasContainingBlock ? 0 : footerHeight;
+  // We don't take into account footer height when the scroll parent is child of document body.
+  // Because in this case, we think the footer is outside the overflow parent.
+  const [hasOverflowParent, setHasOverflowParent] = useState(false);
+  const consideredFooterHeight = hasContainingBlock || hasOverflowParent ? 0 : footerHeight;
```

Comment:
nit: This concatenation of `||` with the ternary operator can be difficult to interpret, maybe we could make it more easy to read?

Reply:
Tried to add parentheses but I guess the linter doesn't like the parentheses here 🤔